### PR TITLE
Add option to add attachments to mail message

### PR DESCRIPTION
**Pull request by sturman**
This pull requests partially implements issue #5 . It's possible to specify multiple file names that will be added as mail attachment.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -48,9 +58,7 @@ async function main() {
             subject: subject,
             text: content_type != "text/html" ? get_body(body) : undefined,
             html: content_type == "text/html" ? get_body(body) : undefined,
-            attachments: attach_file ? {
-                path: attach_file
-            } : undefined
+            attachments: attach_files ? get_attachments(attach_files) : undefined
```
Comment (dawidd6) [inline main.js:61]:
```suggestion
            attachments: attach_files ? attach_files.split(',').map(f => ({path: f.trim()})) : undefined
```

Comment (dawidd6) [reply main.js:61]: If applied, the function `get_attachments` above could be deleted.

**Comment by dawidd6** [comment]
I would rename `attach_files` to just `attachments`. Thoughts?

File: action.yml [hunk 2]
```diff
@@ -33,6 +33,9 @@ inputs:
     description: Content-Type HTTP header (text/html or text/plain)
     required: false
     default: text/plain
+  attach_files:
+    description: File paths (separated with comma) that will be added to mail message attachments
```
Comment (dawidd6) [inline action.yml:37]:
```suggestion
    description: Files that will be added to mail message attachments (separated with comma)
```

File: main.js [hunk 3]
```diff
@@ -19,6 +19,16 @@ function get_from(from, username) {
...
+function get_attachments(attach_files) {
+    attach_files = attach_files.split(',')
+    let attachments = []
+    for (const attach_file of attach_files) {
+        attachments.push({path: attach_file})
+    }
+
+    return attachments
+}
+
```
Comment (dawidd6) [inline main.js:31]:
```suggestion
```

**Comment by dawidd6** [comment]
I'll update the workflow to test sending with attachments later.

Thanks for your contribution!

**Comment by Anshita-Bhasin** [comment]
Can we send artifacts zipped package as attachments over the email using dawid GitHub Actions

**Comment by nileshkr1113** [comment]
How can we attach logs of github action and excel  as an attachment in email

**Pull request merged**